src/test/java/com/endava/internship/service/UserServiceImplTest.java

```diff
@@ -26,7 +26,7 @@
     private static final List<Privilege> ALL_PRIVILEGES = asList(Privilege.values());
 
     @Test
-    void shouldReturnFirstNamesSortedDesc() {
+    public void shouldReturnFirstNamesSortedDesc() {
```

Comment:
why ? For tests, access modifiers must be package-private

Reply:
fixed